src/edu/wright/cs/raiderplanner/controller/MenuController.java

```diff
@@ -1406,6 +1406,15 @@ public void importFile() {
 		this.main();
 	}
 
+	/**
+	 * Handles the 'Settings' Event
+	 * Author: Clayton D. Terrill  1/29/2018
```

Comment:
There is no need to specify that you are  the author of a particular method.

Reply:
Accidently left that in. That is a standard I am used to and did that early on.

Reply:
Yeah, with code in Git, you can just execute `git blame` on a file to see who changed stuff and when. Eclipse also has a way to display that information.